### ISSUE TITLE
Reply to users that request

### ISSUE THREAD

**Issue by JshGrn** [LetsGoDude/OverseerrRequestViaTelegramBot#3]
Once the request has been made, it would be good if the bot would then reply to the user to say the media is now available

**Comment by LetsGoDude** [comment]
I will take a look at how this can be implemented.

**Comment by LetsGoDude** [comment]
Will be implemented after #2 - probably planned for version 2.6 / 2.7

**Comment by LetsGoDude** [comment]
Will be implemented with version 3.0 which will be released in the next few days.
To implement the feature I need the JobQueue extension from PTB, therefore python-telegram-bot[job-queue] has to be installed.

**Comment by mal789** [comment]
With version [v2.5.0] adding users, Overseerr can already notify the user if the admin or user sets up telegram notifications for that specific user.

**Comment by LetsGoDude** [comment]
> With version [v2.5.0] adding users, Overseerr can already notify the user if the admin or user sets up telegram notifications for that specific user.

Yes, that's right. Damn, I've already developed this so far... I'll see if it's possible to set the settings via the bot and automatically transfer the chat ID so that the user has no configuration effort.

**Comment by LetsGoDude** [comment]
Telegram notification settings are now automatically activated with version 2.6.0
